app/services/mailer.js

```diff
@@ -0,0 +1,39 @@
+const nodemailer = require('nodemailer');
+
+const { info } = require('../logger');
+
+const {
+  common: { mailer }
+} = require('../../config');
+const { externalServiceError } = require('../errors');
+
+let transporter = null;
+
+const getTransporter = () => {
+  if (!transporter) {
+    transporter = nodemailer.createTransport({
+      host: mailer.host,
+      port: mailer.port,
+      secure: false,
+      auth: {
+        user: mailer.user,
+        pass: mailer.pass
+      }
+    });
+  }
+  return transporter;
+};
```

Comment:
Correct me if I'm wrong, but this is set only once. Assign it directly to the transporter variable..

Reply:
I can't because if i set in a const, then i can't mock transporter on tests